### DISCUSSION TITLE
Change the extra-manifests folder

### DISCUSSION THREAD

**Discussion by Semptic** [mysticaltech/terraform-hcloud-kube-hetzner#943] [Q&A] [answered]
Hi Guys,

I currently have following setup:

```
├── cluster
│   ├── kube
│   │   └── main.tf
│   └── main.tf
├── extra-manifests
│   └──kustomization.yaml.tpl
├── main.tf
├── services
```

I would like the move the `extra-manifests` folder into the `cluster/kube/` subfolder where also the kube-hetzner module lives. Is there a way to achieve this?

**Comment by mysticaltech** [comment, marked as answer]
@Semptic Currently we have no way of doing that, but it shouldn't be a hard feature to add via another variable for the path of extra-manifests, please feel free to submit a PR. Also, make sure the default value is ./ so that the change is backward compatible.
